app/controllers/content_items_controller.rb

```diff
@@ -10,32 +10,26 @@ def show
       PublishIntent.find_by_path(encoded_request_path)
     end
 
-    scheduled_publishing_log = GovukStatsd.time('show.find_scheduled_publishing_log_entry') do
-      ScheduledPublishingLogEntry.latest_with_path(encoded_request_path)
-    end
-
     set_cache_headers(item, intent)
 
     return error_404 unless item
     return redirect_canonical(item) if item.base_path != encoded_request_path
 
     if can_view(item)
-      render(
-        json: ContentItemPresenter.new(item, api_url_method, scheduled_publishing: scheduled_publishing_log),
-        status: http_status(item)
-      )
+      render json: ContentItemPresenter.new(item, api_url_method), status: http_status(item)
     else
       render json_forbidden_response
     end
   end
 
   def update
+    intent = PublishIntent.find_by_path(encoded_base_path)
+    log_entry = find_or_create_scheduled_publishing_log(encoded_base_path, @request_data["document_type"], intent)
```

Comment:
As far as I can see, for most content, which I assume isn't published with a prior intent, this will be `nil`? If that is the case, it might be good to incorporate that in to the name somehow?

Reply:
I'm struggling to come up with a non-clunky name. The best options so far are `optional_scheduled_publishing_log` or `optional_publishing_log`, but I'm worried it implies that you can opt into creating the log, when really it gets created for a specific type of document.

Any better suggestions?

Reply:
The only suggestion I have would be to make the name more obscure, e.g. something like `scheduled_publishing_log_entry`, which at least prevents any assumptions being made about what the method does from the name.

I'm quite happy to go with the existing name though.